iocore/net/P_SSLUtils.h

```diff
@@ -215,6 +217,25 @@ namespace detail
 /* namespace ssl */ } /* namespace detail */
 }
 
+struct ats_wildcard_matcher {
```

Comment:
yah and also this should have been in the header file so I could access these from SSLSNIConfig